### DISCUSSION TITLE
No result Fallback

### DISCUSSION THREAD

**Discussion by onesounds** [Flow-Launcher/Flow.Launcher#853] [Ideas]
![fabllack1](https://user-images.githubusercontent.com/6903107/144358580-b62df53f-496c-4c4d-9938-63aa08db4f0e.png)
![fallback1](https://user-images.githubusercontent.com/6903107/144358615-460405df-3fe3-404a-b4a4-a7b12b31ac5d.png)

in now, display nothing when there are no result

**Plan**
- Add "Fallbacks" Setting (Where?)
- Three options. 1. nothing 2. Display "No result" 3. Fallback
- or two options. 1. display "no result" 2. Fallback
- Fallback is Default
- Fallback can setting web search sources. (I think no other plug-in is needed.)
- Several can be set.
- This function should work even if the web search plug-in is turned off.

**Matters to consider**
- I think it's appropriate to put this function on a web search.
- put in "fallback" checkbox in websearch source items.
- users do not want to use the web search plug-in, but there are cases where they only want to use the pole. In this case, since the user can set the action keyword, it is not a big problem. (In fact, this function does not seem necessary except in this case. So this work is of low importance.)

![image](https://user-images.githubusercontent.com/6903107/144359674-ac7fd1c7-5dee-4fe5-b7cd-80459a1622ce.png)

- How to order?
- If other functions are also needed, it seems that this function should be manufactured as a separate plug-in.

**Comment by jjw24** [comment]
> Add "Fallbacks" Setting (Where?)

In general section of Settings?

> or two options. 1. display "no result" 2. Fallback

having just two might be sufficient, because i think adding a "No results found" is quite user friendly

I suppose if user sets the fallback as a web search result, then we would request them to turn WebSearch plugin on if switched off.
